cmd/helm/inspect.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/helm"
+)
+
+const inspectDesc = `
+This command inspects a chart (directory, file, or URL) and displays information.
+
+By default, inspect will display the 'values.yaml' file in the chart, making it
+easy for chart users to learn about the chart. Other options provide different
```

Comment:
I'm not sure it makes sense to point out other options if there aren't any in this first iteration